src/status_im/ui/screens/wallet/refresh_control.cljs

```diff
@@ -0,0 +1,11 @@
+(ns status-im.ui.screens.wallet.refresh-control
+  (:require [re-frame.core :as re-frame]
+            [reagent.core :as reagent]
+            [quo.react-native :as rn]))
+
+(defn refresh-control
+  []
+  (let [prices-loading? @(re-frame/subscribe [:prices-loading?])]
+    (reagent/as-element
+     [rn/refresh-control {:refreshing prices-loading?
+                          :on-refresh #(re-frame/dispatch [:wallet.ui/pull-to-refresh])}])))
```

Comment:
@rasom hi, I have a question about this event `:wallet.ui/pull-to-refresh`.

For this task, is this a sufficient/correct place to call it?

Because under a single wallet view, there are **Collectibles** and **History**.

So for `:wallet.transactions.history/screen` which populates **History** and for `:wallet/visible-assets-with-values`, which would deal with **Assets** & **Collectibles**, should I do any additional work? And if I should, is there a way to add this data, as a stub to pre-populate it, so that it's meaningful?

Thanks
/cc @hesterbruikman

Reply:
For context here is the view I'm talking about:


<img width="535" alt="Screenshot 2020-08-26 at 13 57 08" src="https://user-images.githubusercontent.com/152363/91306201-bfadb180-e79b-11ea-8d99-057c86e43733.png">